Website/src/rendering/src/components/SpeakerInformationPageHero.tsx

```diff
@@ -65,13 +66,19 @@ const SpeakerInformationPageHero = (props: SpeakerInformationPageHeroProps): JSX
     ) : undefined;
 
   return (
-    <InformationPageHero
-      {...propsRest}
-      fields={newFields}
-      type="speaker"
-      qualificative={qualificative}
-      informations={informations}
-    />
+    <>
+      <Head>
+        <meta name="FeaturedSpeaker" content={props.fields.Featured.value ? 'true' : 'false'} />
```

Comment:
Why was this meta added? Is there a CDP/Personalize scenario using it?

Reply:
other featured pages has it, like premium speaker. so I added it.